src/varname.jl

```diff
@@ -1,5 +1,6 @@
 using Accessors
 using Accessors: ComposedOptic, PropertyLens, IndexLens, DynamicIndexLens
+import Accessors: opcompose, ⨟
```

Comment:
This should be unnecessary, Accessors reexports them (they are defined in CompositionsBase). They're aliases, so one of them would be sufficient. Generally, though, I'd avoid `import` since it makes it easier to accidentally extend methods.
```suggestion
```